### PR TITLE
What ho, Muscle Wizard! Might you cast us a spell?

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
martial artist apprentice with plasmafist and the epic superpunch spell because plasmafist actually sucks and doesn't get the aggrograbs that almost every martial art has

## Why It's Good For The Game
i would've added mutate too if i could've gotten away with it

## Changelog
:cl:
add: martial apprentices for the local Chinese wizard
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
i couldn't actually test it for myself outside of the fact it does show up as an option in the contract because >local testing

**Comment by YakumoChen** [comment]
i can only imagine a huge puff of smoke as a bunch of huge muscley men appear, bump their abs together, and then just start punching holes and people through the station

File: code/modules/antagonists/_common/antag_spawner.dm [hunk 1]
```diff
@@ -37,6 +37,8 @@
 		dat += "<I>Your apprentice is training to cast spells that will aid your survival. They know Forcewall and Charge and come with a Staff of Healing.</I><BR>"
 		dat += "<A href='byond://?src=[REF(src)];school=[APPRENTICE_ROBELESS]'>Robeless</A><BR>"
 		dat += "<I>Your apprentice is training to cast spells without their robes. They know Knock and Mindswap.</I><BR>"
+		dat += "<A href='byond://?src=[REF(src)];school=[APPRENTICE_MARTIAL]'>Martial Artist</a><BR>"
+		dat += "<I>Your apprentice is training in ancient martial arts. They know the Plasmafist and Lesser Heal.</I><BR>"
```
Comment (Putnam3145) [inline code/modules/antagonists/_common/antag_spawner.dm]: nuclear fist, not lesser heal

**Review by Putnam3145 — DISMISSED** [review]

File: code/modules/antagonists/_common/antag_spawner.dm (hunk 1 above)
Comment (ghost) [reply code/modules/antagonists/_common/antag_spawner.dm]: oops

**Review by Ghommie — APPROVED** [review]

**Comment by ghost** [comment]
whew

**Pull request merged**